testing/src/cypress/e2e/04-api-institution/02-instition-create-update-location.cy.ts

```diff
@@ -43,13 +47,15 @@ function createOrUpdateInstitutionLocation(
       primaryContactLastName: `Auto-${uniqueId}-LastName`,
       primaryContactEmail: `Auto@${uniqueId}.com`,
       primaryContactPhone: "32165496872",
+      applicationId: "",
```

Comment:
`applicationId` and `applicationStatus` are not needed on the payload. Please remove them.

Reply:
Done